src/eckit/utils/RLE.h

```diff
@@ -26,7 +27,7 @@ namespace eckit {
 class Stream;
 
 template <class InputIterator, class OutputIterator>
```

Comment:
I would do something like, before you use steady_clock and in the header:
using EncodingClock = std::chrono::steady_clock;

And use this type instead of explicitly refering to std::chrono::steady_clock. In this overview, I think you can replace every single instance of referring inside std::chrono with this new type.